Dockerfile

```diff
@@ -36,16 +37,16 @@ COPY ./log4j.properties src/main/resources/log4j.properties
 
 # build and install, placing the scripts jar back in the target folder
 # where import scripts expect it after cleanup
-RUN mvn -DskipTests clean package \
-	&& unzip portal/target/cbioportal-*.war -d $CATALINA_HOME/webapps/cbioportal \
+RUN mvn -DskipTests clean install \
+	&& unzip portal/target/cbioportal.war -d $CATALINA_HOME/webapps/cbioportal \
 	&& mv scripts/target/scripts-*.jar /root/ \
 	&& mvn clean \
 	&& mkdir scripts/target/ \
 	&& mv /root/scripts-*.jar scripts/target/
 
 # add runtime plumbing to Tomcat config:
 # - make cBioPortal honour db config in portal.properties
-RUN echo 'CATALINA_OPTS="-Dauthenticate=false $CATALINA_OPTS -Ddbconnector=dbcp"' >>$CATALINA_HOME/bin/setenv.sh
+RUN echo 'CATALINA_OPTS="-Dauthenticate=false $CATALINA_OPTS -Ddbconnector=dbcp -Dsession.service.url=http://cbio-session-service:8080/api/sessions/main_session/"' >>$CATALINA_HOME/bin/setenv.sh
```

Comment:
I don't think that supplying this here, rather than in the portal.properties config file where I'd expect it, is documented in the cBioPortal deployment manual. But from what you've told me, I believe that you discussed this with the community, and the way to documented  is going to change again fairly soon. Is that correct?

Reply:
If so, it might be worth adding a comment to explain where the information came from. I should have done the same to link the dbconnector parameter to issue https://github.com/cBioPortal/cbioportal/issues/4832.

Reply:
Still no link, but the comment should be good enough.